test/Test/Database/LSMTree/Internal/RawBytes.hs

```diff
@@ -1,11 +1,8 @@
-{- HLINT ignore "Avoid restricted alias" -}
```

Comment:
We shouldn't ignore this.

Reply:
It's removed now, so I'm not sure what you mean

Reply:
Just in case you were moving it to the global file.